### PR TITLE
[Graph Optimization] SOT+CUDAGraph support  ERNIE4.5T VL 28B / 424B

### PR DESCRIPTION
**Pull request by DrRyanHuang**
## Motivation
SOT+CUDAGraph 跑通 ERNIE4.5T VL 28B / 424B
前置PR: https://github.com/PaddlePaddle/FastDeploy/pull/4610

## Modifications

- fastdeploy/model_executor/models/ernie4_5_vl/ernie4_5_vl_moe.py
- 添加 SOT+CUDAGraph 运行 ERNIE4.5T VL 28B 的单测

## Usage or Command

```shell
MODEL=/workspace/EB_MODELS/ERNIE-4.5-VL-28B-A3B-Paddle
MODEL=/workspace/EB_MODELS/ERNIE-4.5-VL-424B-A47B-Paddle
rm -rf log/*

export FLAGS_cuda_graph_blacklist="custom_op.static_op_append_attention_with_output_"
export CUDA_VISIBLE_DEVICES=0,1,2,3
export PORT=39905

python -m fastdeploy.entrypoints.openai.api_server \
  --model $MODEL \
  --metrics-port 39717 \
  --port $PORT \
  --engine-worker-queue-port 39719 \
  --tensor-parallel-size 4 \
  --max-model-len 32768 \
  --max-num-seqs 128 \
  --quantization wint4 \
  --enable-mm \
  --graph-optimization-config '{"graph_opt_level": 1, "use_cudagraph": true, "full_cuda_graph": false}' 
```

## Accuracy Tests
NULL

## Checklist

- [x] Add at least a tag in the PR title.
  - Tag list: [`[FDConfig]`,`[APIServer]`,`[Engine]`, `[Scheduler]`, `[PD Disaggregation]`, `[Executor]`, `[Graph Optimization]`, `[Speculative Decoding]`, `[RL]`, `[Models]`, `[Quantization]`, `[Loader]`, `[OP]`, `[KVCache]`, `[DataProcessor]`, `[BugFix]`, `[Docs]`, `[CI]`, `[Optimization]`, `[Feature]`, `[Benchmark]`, `[Others]`, `[XPU]`, `[HPU]`, `[GCU]`, `[DCU]`, `[Iluvatar]`, `[Metax]`]
  - You can add new tags based on the PR content, but the semantics must be clear.
- [x] Format your code, run `pre-commit` before commit.
- [x] Add unit tests. Please write the reason in this PR if no unit tests.
- [ ] Provide accuracy results.
- [ ] If the current PR is submitting to the `release` branch, make sure the PR has been submitted to the `develop` branch, then cherry-pick it to the `release` branch with the `[Cherry-Pick]` PR tag.

### CODE REVIEW

**Comment by EmmonsCurse** [comment]
You may need to update your branch to resolve errors.

**Review by gongshaotian — DISMISSED** [review]
LGTM

File: fastdeploy/model_executor/models/ernie4_5_vl/ernie4_5_vl_moe.py [hunk 1]
```diff
@@ -277,7 +277,7 @@ def load_state_dict(self, state_dict):
     def forward(self, hidden_states: paddle.Tensor, vl_moe_meta: VLMoEMeta):
         if self.num_shared_experts > 0:
             shared_experts_out = self.shared_experts(hidden_states)
-        hidden_states, vl_moe_meta.text_input, vl_moe_meta.image_input = text_image_gather_scatter(
+        hidden_states, text_input, image_input = text_image_gather_scatter(
```
Comment (gongshaotian) [inline fastdeploy/model_executor/models/ernie4_5_vl/ernie4_5_vl_moe.py:280]: 这里和xpu那个修改会冲突吗

Comment (DrRyanHuang) [reply fastdeploy/model_executor/models/ernie4_5_vl/ernie4_5_vl_moe.py:280]: XPU 的 https://github.com/PaddlePaddle/FastDeploy/pull/4636 合入后，这个再合入就没问题

**Review by SigureMo — APPROVED** [review]

**Review by EmmonsCurse — APPROVED** [review]

**Review by gongshaotian — COMMENTED** [review]
LGTM

**Review by gongshaotian — APPROVED** [review]
